common/assets/sass/application.scss

```diff
@@ -56,6 +56,7 @@ p {
 
 .extra-error {
   margin-top: -18px;
+  margin-bottom: 25px;
```

Comment:
We should go through and replace allot of these pixel values with the GDS constants.